.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install bats
+      - name: Set up Git user
+        run: |
+          git config --local user.name "${GITHUB_ACTOR}"
+          git config --local user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+      - name: Set up env vars
+        run: echo "CURRENT_GIT_REF='$(echo ${GITHUB_REF} | awk 'BEGIN { FS = "/" } ; { print $3 }')'" >> "${GITHUB_ENV}" # Pass the tag, or target branch for a PR, or the branch
```

Comment:
What is the rationale behind this code?
WDYT about moving it into the Makefile to provide as much reusability as possible?

Reply:
Current Makefile works with `CURRENT_GIT_REF` variable that it either gets externally or calculates via `git rev-parse --abbrev-ref HEAD` that returns things like `master`.

On Travis, there is `TRAVIS_BRANCH` variable that is weirdly named (because it can point to both branches and tags), but is fully equivalent to format output by `git rev-parse --abbrev-ref HEAD`.

Now, on GitHub Actions, there's no direct equivalent. GitHub Actions has `GITHUB_REF` variable, that has values like `refs/heads/master` instead of just `master`. So we need to strip `refs/heads` (or `refs/tags`) part, that's what that code does. Alternatively, I can change `CURRENT_GIT_REF` Makefile code to stop passing `--abbrev-rev`, so Makefile always works with non-abbreviated string and cuts `refs/heads`/`refs/tags` part itself.

Reply:
Thanks for the explanation! Yes it would be cool to move this logic to the Makefile to uncouple a bit more from the CI system. Does it look good?